src/test/groovy/com/github/vkorobkov/jfixtures/util/YmlUtilTest.groovy

```diff
@@ -39,6 +39,42 @@ class YmlUtilTest extends Specification implements YamlVirtualFolder {
         yaml.isEmpty()
     }
 
+    def "should not have yml twin"() {
```

Comment:
Let's add method name in the beginning of the test method name:
`def "#hasYamlTwin does not have twin for .yml file"`